oscore-cf/pom.xml

```diff
@@ -0,0 +1,35 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.eclipse.californium</groupId>
+    <artifactId>parent</artifactId>
+    <version>2.0.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>oscore-cf</artifactId>
+  <properties>
+  	<cose.version>0.9.7</cose.version>
+  	<cbor.version>3.0.3</cbor.version>
+  	<numbers.version>1.0.2</numbers.version>
+  </properties>
+  <dependencies>
+	<dependency>
+		<groupId>com.augustcellars.cose</groupId>
+		<artifactId>cose-java</artifactId>
+		<version>${cose.version}</version>
+	</dependency>
+	<dependency>
+		<groupId>org.eclipse.californium</groupId>
+		<artifactId>scandium</artifactId>
+		</dependency>
```

Comment:
Please, format the pOM also proper, the indention  of the closing element seems to be wrong.
I miss the dependency to "californium-core" (and my eclipse ide and maven miss it as well :-) ).
Also the test dependencies are missing

```
		<!-- test dependencies -->
		<dependency>
			<groupId>junit</groupId>
			<artifactId>junit</artifactId>
			<scope>test</scope>
		</dependency>
		<dependency>
			<groupId>ch.qos.logback</groupId>
			<artifactId>logback-classic</artifactId>
			<scope>test</scope>
		</dependency>
```

I'm wondering, that this works on your machine. Red crosses in my eclipse IDE and errors in maven. So does this really build on your machine?

Reply:
It worked fine on my end. I will format correctly and add the test dependencies.